### PR TITLE
Replaced cp command with patch for MailDependencyProvider.php

### PR DESCRIPTION
**Pull request by konstantin-tishkin-spryker**

### CODE REVIEW

File: setup_suite.sh [hunk 1]
```diff
@@ -25,7 +25,8 @@ cp -r ${APPLICATION_PATH}/public/Zed/maintenance /maintenance
 # Swift Mailer AWS configuration
 if [ $SMTP_HOST != "127.0.0.1" ]
      then
-        cp /etc/spryker/MailDependencyProvider.php src/Pyz/Zed/Mail/MailDependencyProvider.php
+#        cp /etc/spryker/MailDependencyProvider.php src/Pyz/Zed/Mail/MailDependencyProvider.php
+        patch -p0  ./src/Pyz/Zed/Mail/MailDependencyProvider.php < /etc/spryker/Mailer.patch
```
Comment (vvs82) [inline setup_suite.sh]: Please check if the file `MailDependenceProvider.php` exists.

File: application/app_files/Mailer.patch [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
++
++     protected function addMailer(Container $container)
++     {
++         $container[static::MAILER] = function () {
++             $message = new Swift_Message();
++             $transport = new Swift_SmtpTransport(
++                 $this->getConfig()->getSmtpHost(),
++                 $this->getConfig()->getSmtpPort()
++             );
++               $transport->setHost(getenv('SMTP_HOST'));
```
Comment (vvs82) [inline application/app_files/Mailer.patch]: Delete space please.

**Comment by vvs82** [comment]
👍

File: application/app_files/Mailer.patch (hunk 2 above)
Comment (konstantin-tishkin-spryker) [reply application/app_files/Mailer.patch]: done

**Pull request merged**